mods/d2k/weapons.yaml

```diff
@@ -66,19 +67,14 @@ Sniper:
 	Warhead@1Dam: SpreadDamage
 		Spread: 32
 		Damage: 60
-		Versus:
-			None: 100
-			Wood: 0
-			Light: 1
-			Heavy: 0
-			Concrete: 0
+		ValidTargets: Infantry
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
 
 Vulcan:
 	ReloadDelay: 30
 	Range: 5c768
 	Report: VULCAN.AUD
-	ValidTargets: Ground
+	InvalidTargets: Structure
```

Comment:
`ValidTargets` defaults to `Ground` and `Water`, you'll find that many normal weapons don't explicitly define `ValidTargets` at all, so that entry was simply redundant (though you could argue it removed `Water` since that doesn't exist in d2k).